### ISSUE TITLE
Enhancement: Update the message create webhook event to include more details

### ISSUE THREAD

**Issue by pranavrajs** [chatwoot/chatwoot#518]
Current webhook event payload does not contain much information about the conversation and the inbox. This is an enhancement request to update the payload of the webhook event to match the following.

```js
{
  type: "message_create",
  id: "message_id"
  content: "The content of the message",
  created_at: "time",
  message_type: "incoming/outgoing/template/activity",
  source_id: "If message inbox is twitter or facebook",
  sender: { // If message is sent by an agent
    id: "agent_id",
    name: "name of the agent",
    email: "email of the agent",
  },
  contact: { // If message is sent by a contact 
    id: "contact_id"
    name: "name_of_contact"
  },
  inbox: {
    id: "inbox_id",
    name: "Name of the inbox"
  },
  conversation: {
    display_id: "id_of_the_conversation",
    additional_attributes: {}
  },
  account: {
    id: "account_id",
    name: "account_name"
  }
}
```

**Comment by alfredfrancis** [comment]
Is there any documentation on how to use the webhooks feature? I couldn't find anything in the Admin Console.

**Comment by sojan-official** [comment]
@alfredfrancis  we are adding this to the admin console . have a look at https://github.com/chatwoot/chatwoot/issues/493

**Comment by pranavrajs** [comment]
Closed via #562
